ToDoList_Project/ToDoList_Project/Controllers/UsersController.cs

```diff
@@ -0,0 +1,64 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using ToDoList_Project.DTOs;
+using ToDoList_Project.Models;
+using ToDoList_Project.Repository;
+
+namespace ToDoList_Project.Controllers
+{
+    [Route("api/[controller]")]
+    [ApiController]
+    public class UsersController : ControllerBase
+    {
+        private readonly IUsersRepository usersRepository;
+
+        public UsersController(IUsersRepository usersRepository)
+        {
+            this.usersRepository = usersRepository;
+        }
+
+        [HttpPost("register")]
+        public async Task<object> SignUp(Register userRegister)
+        {
+            if (await usersRepository.SignUp(userRegister) == null)
+            {
+                return Ok(new Respond<object>
+                {
+                    IsSuccess = false,
+                    Message = "Sign up fail!",
+                    Result = userRegister,
+                });
+            }
+            else
+            {
+                return Ok(new Respond<object>
+                {
+                    IsSuccess = true,
+                    Message = "Sign up success!",
+                    Result = userRegister,
+                });
+            }
+        }
+
+        [HttpPost("login")]
+        public async Task<object> Login(Login user)
+        {
+            var Token = await usersRepository.Login(user);
+            if(Token == null)
+            {
+                return BadRequest(500);
```

Comment:
```suggestion
                return BadRequest();
```
đã BadReqeuest rồi lại còn 500 nữa :)))